components/admin_console/admin_definition.jsx

```diff
@@ -6886,7 +6886,10 @@ const AdminDefinition = {
                         help_text: t('admin.experimental.enableSharedChannels.desc'),
                         help_text_default: 'Toggles Shared Channels',
                         help_text_markdown: false,
-                        isHidden: it.not(it.licensedForFeature('SharedChannels')),
+                        isHidden: it.not(it.any(
+                            it.licensedForSku(LicenseSkus.Enterprise),
+                            it.licensedForSku(LicenseSkus.Professional),
+                        )),
```

Comment:
I think we need to check both the feature and the license SKU. I've pinged someone on the growth/self-serve team on the server PR - https://github.com/mattermost/mattermost-server/pull/21882#issuecomment-1362317834

Reply:
After the discussions there, I've also re-added the check for the feature itself, so it should cover all cases now.